### PR TITLE
add people from data

### PR DESCRIPTION
**Pull request by AnnSerdechna**
[Demo](https://annserdechna.github.io/react_people-table-basics/)

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import App from './App';
+import { HomePage } from './components/HomePage';
+import { NotFoundPage } from './components/NotFoundPage';
+import { PeoplePage } from './components/PeoplePage';
+
+const rootElement = document.getElementById('root');
+
+render(
+  <BrowserRouter>
```
Comment (mariiishka) [inline src/index.tsx:11]: change to HashRouter

File: src/components/PeoplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { FC, useState, useEffect } from 'react';
+import { getPeople } from '../data';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: FC = () => {
```
Comment (mariiishka) [inline src/components/PeoplePage.tsx:5]: Wrap components with React.memo

Comment (mariiishka) [inline src/components/PeoplePage.tsx:5]: Wrap components with React.memo

File: src/components/PeopleTable.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    <tbody>
+      {people.map((user) => (
+        <tr key={user.slug}>
+          <td>{user.name}</td>
+          <td>{user.sex}</td>
+          <td>{user.born}</td>
+          <td>{user.died}</td>
+          <td>{user.fatherName}</td>
+          <td>{user.motherName}</td>
+        </tr>
```
Comment (mariiishka) [inline src/components/PeopleTable.tsx:29]: create a PersonRow component
Снимок

Comment (mariiishka) [inline src/components/PeopleTable.tsx:29]: create a PersonRow component
Снимок

**Review by mariiishka — CHANGES_REQUESTED** [review]
create a separate folder for each component

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**